python/pyspark/ml/param/__init__.py

```diff
@@ -15,11 +15,24 @@
 # limitations under the License.
 #
 
+import uuid
 from abc import ABCMeta
 
-from pyspark.ml.util import Identifiable
+__all__ = ['Param', 'Params']
 
-__all__ = ["Param"]
+
+class Identifiable(object):
```

Comment:
Should we put `Identifiable` under `param`? I think it should either under `ml` or `ml.util`.